### ISSUE TITLE
Imports not working correctly

### ISSUE THREAD

**Issue by RafaelVidaurre** [nathantreid/meteor-css-modules#57]
I'm using `francocatena:compass` package, and when trying to import the library on a `.scss` file I get this:

```
[Error: File '/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset' not found at any of the following paths: ["/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset.scss","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset.sass","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/_reset","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/_reset.scss","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/_reset.sass"]]
[Error: File '/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset' not found at any of the following paths: ["/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset.scss","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/reset.sass","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/_reset","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/_reset.scss","/Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/francocatena_compass/compass/_reset.sass"]]
=> Errors prevented startup:                  

   While processing files with nathantreid:css-modules (for target web.browser):
   /Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/input.js:61:22:
   /Users/rafaelvidaurre/Code/pathable/pathable-admin/packages/pathable-styles/imports/host.scss:3:1: Unknown word
   at Input.error
   (/Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/input.js:61:22)
   at Parser.unknownWord
   (/Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/parser.js:457:26)
   at Parser.word
   (/Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/parser.js:174:14)
   at Parser.loop
   (/Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/parser.js:60:26)
   at parse (/Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/parse.js:26:12)
   at new LazyResult
   (/Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/lazy-result.js:61:24)
   at Processor.process
   (/Users/rafaelvidaurre/.meteor/packages/nathantreid_css-modules/.2.1.0.1alj0r2++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/mss/node_modules/postcss/lib/processor.js:34:16)
   at CssModulesProcessor.load (packages/mss/css-modules-processor.js:74:5)
   at packages/mss/css-modules-processor.js:33:10
   at tryCallTwo (/Users/rafaelvidaurre/.meteor/packages/meteor-tool/.1.3.4_1.pa525p++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/promise/lib/core.js:45:5)
   at doResolve (/Users/rafaelvidaurre/.meteor/packages/meteor-tool/.1.3.4_1.pa525p++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/promise/lib/core.js:200:13)
   at new Promise (/Users/rafaelvidaurre/.meteor/packages/meteor-tool/.1.3.4_1.pa525p++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/promise/lib/core.js:66:3)
   at CssModulesProcessor.processInternal (packages/mss/css-modules-processor.js:28:11)
   at CssModulesProcessor.process (packages/mss/css-modules-processor.js:18:26)
   at CssModulesBuildPlugin.processFile (packages/mss/css-modules-build-plugin.js:247:22)
   at Array.forEach (native)
   at CssModulesBuildPlugin.compileCssModules (packages/mss/css-modules-build-plugin.js:239:34)
   at CssModulesBuildPlugin.processFilesForTarget (packages/mss/css-modules-build-plugin.js:62:21)


=> Your application has errors. Waiting for file change.
```

Have not been able to debug it. It looks like it is getting the paths incorrectly or something

**Comment by akanix42** [comment]
Can you post a repro on github so I can see exactly what you're seeing? Thanks.

**Comment by RafaelVidaurre** [comment]
Sure, I'll take some time to create a repo for that

**Comment by RafaelVidaurre** [comment]
@nathantreid Okay, I managed to fix some parts of my issue while doing the example repo. One thing to note is that the errors thrown when `node-sass` is not installed are not very helpful.

Okay so what, apparently IS a bug is the following error:
<img width="709" alt="screen shot 2016-07-14 at 1 59 41 pm" src="https://cloud.githubusercontent.com/assets/1808674/16850058/4a353f1c-49cb-11e6-90dc-0a837f583593.png">

This error _only_ shows on the client. The app is able to find the file when loading, but on the browser we do get the error.

This is the repo: https://github.com/Narzerus/css-modules-bug-example

There's two of them, one is for the package from which we import the file.

**Comment by RafaelVidaurre** [comment]
Any news on this @nathantreid ? Cheers

**Comment by flippyhead** [comment]
I'm curious about this too.

**Comment by akanix42** [comment]
This is caused by a bug in my code when importing package files. I've got a fix for it - I just need to perform a little more testing to make sure it doesn't break anything.
I expect to have it out tomorrow.

**Comment by crackedbeefcake** [comment]
is this fixed? because i have the same error...
`install.js: 101 Uncaught Error: Cannot find module './styles/new.sass'`

**Comment by crackedbeefcake** [comment]
out of nowhere. no updates or relevant code changes..

**Comment by RafaelVidaurre** [comment]
@nathantreid this is fix is not up yet right?

**Comment by akanix42** [comment]
@Narzerus This has just now been released in v2.1.1.
